content/scout/policy/ci.md

```diff
@@ -0,0 +1,141 @@
+
+This example doesn't assume that you're using Docker Hub as your container
+registry. As a result, this workflow uses the `docker/login-action` twice:
+
+- Once for authenticating to your container registry.
+- Once more for authenticating to Docker to pull the analysis results of your
+  `production` image.
+
+If you use Docker Hub as your container registry, you only need to authenticate
+once.
+
+For the policy evaluation to work, you must make sure that you load the image
+to the local image store. Note that this means you can't run this workflow with
+a multi-platform image, and you can't build the image with attestations. That's
+because the local image store doesn't currently support loading such images.
+
+Also note the `pull-requests: write` permission for the job. The Docker Scout
+GitHub Action adds a pull request comment with the evaluation results by
+default, which requires this permission. For details, see
+[Pull Request Comments](https://github.com/docker/scout-action#pull-request-comments).
+
+```yaml
+name: Docker
+
+on:
+  push:
+    tags: ["*"]
+    branches:
+      - "main"
+  pull_request:
+    branches: ["**"]
+
+env:
+  REGISTRY: docker.io
+  IMAGE_NAME: <IMAGE_NAME>
+  DOCKER_ORG: <ORG>
+
+jobs:
+  build:
+    permissions:
+      pull-requests: write
+
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Setup Docker buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Log into registry ${{ env.REGISTRY }}
+        uses: docker/login-action@v3
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ secrets.REGISTRY_USER }}
+          password: ${{ secrets.REGISTRY_TOKEN }}
+
+      - name: Extract metadata
+        id: meta
+        uses: docker/metadata-action@v4
+        with:
+          images: ${{ env.IMAGE_NAME }}
+
+      - name: Build image
+        id: build-and-push
+        uses: docker/build-push-action@v4
+        with:
+          context: .
+          tags: ${{ steps.meta.outputs.tags }}
+          labels: ${{ steps.meta.outputs.labels }}
+          sbom: ${{ github.event_name != 'pull_request' }}
+          provenance: ${{ github.event_name != 'pull_request' }}
+          push: ${{ github.event_name != 'pull_request' }}
+          load: ${{ github.event_name == 'pull_request' }}
+
+      - name: Authenticate with Docker
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKER_USER }}
+          password: ${{ secrets.DOCKER_PAT }}
+
+      - name: Compare
+        if: ${{ github.event_name == 'pull_request' }}
+        uses: docker/scout-action@v1
+        with:
+          command: compare
+          image: ${{ steps.meta.outputs.tags }}
+          to-env: production
+          platform: "linux/amd64"
+          ignore-unchanged: true
+          only-severities: critical,high
+          organization: ${{ env.DOCKER_ORG }}
+          exit-on: policy
+```
```

Comment:
It'd be great to have a screenshot of the PR's comment showing the policy getting worse or better.